xwiki-platform-core/xwiki-platform-mail/xwiki-platform-mail-send/xwiki-platform-mail-send-storage/src/main/java/org/xwiki/mail/internal/DatabaseMailResender.java

```diff
@@ -165,6 +165,14 @@ private MailStatusResult resend(String batchId, String uniqueMessageId) throws M
     @FunctionalInterface
     private interface SingleMailResender
```

Comment:
@AkMo3 I don't understand why Checkstyle complains about the public method below since it's in a private class, i.e this class is not supposed to be used by API users. What's the rationale and is and there a config for this? Thx

Reply:
This was the reason for making this PR draft. I am working on it actually. @nmancus1 What do think of this one?

Reply:
Fixed. There are no more being reported. Thanks for heads up!